config/locales/candidate_interface/gcse.yml

```
@@ -142,3 +142,13 @@ en:
               blank: Enter your UK ENIC reference number
             comparable_uk_qualification:
               blank: Choose a comparable UK qualification
+        candidate_interface/gcse_not_completed_form:
+          attributes:
+            currently_completing_qualification:
+              inclusion: Select yes if you are currently studying for a GCSE in %{subject}, or equivalent
+            not_completed_explanation:
+              blank: Enter details of the qualification you’re studying for
```

Can we customise the over word count error slightly?

```suggestion
            not_completed_explanation:
              blank: Enter details of the qualification you’re studying for
              too_many_words: Qualification details must be %{count} words or fewer              
```